docs/api.md

```diff
@@ -1361,7 +1361,7 @@ most.from([1, 2, 3, 4])
 ### switch
 
 ####`stream.switch() -> Stream`
-####`most.switch(stream) -> Stream`
+####`most.switchLatest(stream) -> Stream`
```

Comment:
@briancavalier If all say, 'Go!', I’ll make these doc changes as you mentioned.

@TylorS @davidchase